library-best-practices/glossary.md

```diff
@@ -66,3 +66,72 @@ Java Dependency Glossary
 - **Linkage-matchable version** (said of a particular version of A in relation
   to all versions of B): There exists some version of B such that the version of
   A and the version of B are linkage-compatible.
+
+
+### Static Linkage Check
```

Comment:
```suggestion
### Static Linkage Checking
```

Reply:
I feel like we should be defining states here; that is, static linkage and static linkage errors; not processes such as a check